src/System.Private.Xml/tests/XmlSerializer/ReflectionOnly/System.Xml.XmlSerializer.ReflectionOnly.Tests.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);ReflectionOnly</DefineConstants>
     <ProjectGuid>{4050F1D1-1DD2-4B48-A17B-E3F955518C4B}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>  
   </PropertyGroup>
```

Comment:
It is `#if uap` now, but if we can get rid of it all together, it will be even better :). Not sure why it's there.

---
In reply to: [105093279](https://github.com/dotnet/corefx/pull/16704#discussion_r105093279) [](ancestors = 105093279)